src/Http/routes.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
 Route::group(['prefix' => 'pages', 'namespace' => 'Ourgarage\StaticPages\Http\Controllers'], function () {
-    Route::get('/', 'StaticPagesController@page_list');
-    Route::get('/{slug}', 'StaticPagesController@page_view');
+    Route::get('/', 'StaticPagesController@page_list')->name('page-list');
```

Comment:
let's give the names with some prefix. For example like `static-pages::page-list`.